examples/data-sources/genesyscloud_routing_email_domain/data-source.tf

```diff
@@ -1,4 +1,4 @@
 data "genesyscloud_routing_email_domain" "email_domain" {
```

Comment:
Just realised you'll have to run `go generate` to generate files in the docs directory that will show how to use the data source and resource by combining their .tf file and apis.md files

Reply:
Added doc file